bears/js/JSONFormatBear.py

```diff
@@ -34,6 +34,13 @@ def run(self, filename, file,
         :param escape_unicode: Whether or not to escape unicode values using
```

Comment:
Ah one last: Please remove the "Closes XXXX" reference in the commit message, it does not really apply :)